network/controller/bootstrap/bootstrap.go

```diff
@@ -349,7 +351,14 @@ func (bc *Bootstrap) getDiscoveryNodesChannel(ctx context.Context, discoveryNode
 				inslogger.FromContext(ctx).Errorf("Error bootstrapping to address %s: %s", address, err.Error())
 				return
 			}
-			bootstrapResults <- bootstrapResult
+
+			select {
+			case bootstrapResults <- bootstrapResult:
```

Comment:
horror place. What happens with sender of this message?

Reply:
in this case it's ok, buffered channel used